wikimedia/wikipedia-ios
# AALD: highlight history cells, allow for multiple cells to be highlighted

**Fixes Phabricator ticket:** https://phabricator.wikimedia.org/T266370


### Test Steps
1. In AALD modal, tap various "View changes" buttons. Ensure the appropriate cell (or cells, if multiple small edits were made) are temporarily highlighted.
